recipes/generate_sql_and_execute/generate_sql_and_exec.ipynb

```diff
@@ -0,0 +1,482 @@
+    "    price DECIMAL(10, 2) NOT NULL,\n",
+    "    category VARCHAR(50),\n",
+    "    image_url VARCHAR(255),\n",
+    "    available_quantity INT\n",
+    ");\n",
+    "\n",
+    "CREATE TABLE Store (\n",
+    "    store_id INT NOT NULL PRIMARY KEY,\n",
+    "    name VARCHAR(100) NOT NULL,\n",
+    "    location VARCHAR(255),\n",
+    "    capacity INT,\n",
+    "    available_capacity INT\n",
+    ");\n",
+    "\n",
+    "CREATE TABLE Order (\n",
+    "    order_id INT NOT NULL PRIMARY KEY,\n",
+    "    user_id INT REFERENCES User(user_id),\n",
+    "    order_date TIMESTAMP,\n",
+    "    status VARCHAR(50),\n",
+    "    total_amount DECIMAL(10, 2),\n",
+    "    store_id INT REFERENCES Store(store_id)\n",
+    ");\n",
+    "\n",
+    "CREATE TABLE OrderItem (\n",
+    "    order_item_id INT NOT NULL PRIMARY KEY,\n",
+    "    order_id INT REFERENCES Order(order_id),\n",
+    "    product_id INT REFERENCES Product(product_id),\n",
+    "    quantity INT,\n",
+    "    subtotal DECIMAL(10, 2)\n",
+    ");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f57fad0-986f-4f7a-8f09-247f275792e8",
+   "metadata": {},
+   "source": [
+    "#### Populate the tables with data by executing the following SQL in the DB2 UI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2fe952d-5aa9-4411-bcd3-48febb85555b",
+   "metadata": {},
+   "source": [
+    "INSERT INTO User (user_id, username, email, password_hash, full_name, address, phone_number)\n",
+    "VALUES\n",
+    "    (1, 'john_doe', 'john@example.com', 'hashed_password', 'John Doe', '123 Main St', '123-456-7890'),\n",
+    "    (2, 'jane_smith', 'jane@example.com', 'hashed_password', 'Jane Smith', '456 Elm St', '987-654-3210');\n",
+    "\n",
+    "INSERT INTO Product (product_id, name, description, price, category, image_url, available_quantity)\n",
+    "VALUES\n",
+    "    (1, 'Toy Car', 'Remote control toy car', 19.99, 'Toys', 'car.jpg', 50),\n",
+    "    (2, 'Action Figure', 'Superhero action figure', 12.99, 'Toys', 'action_figure.jpg', 30),\n",
+    "    (3, 'Laptop', 'High-performance laptop', 899.99, 'Electronics', 'laptop.jpg', 10),\n",
+    "    (4, 'Smartphone', 'Latest smartphone model', 699.99, 'Electronics', 'smartphone.jpg', 15);\n",
+    "\n",
+    "INSERT INTO Store (store_id, name, location, capacity, available_capacity)\n",
+    "VALUES\n",
+    "    (1, 'Center A', 'Location A', 100, 80),\n",
+    "    (2, 'Center B', 'Location B', 150, 120);\n",
+    "\n",
+    "INSERT INTO Order (order_id, user_id, order_date, status, total_amount, store_id)\n",
+    "VALUES\n",
+    "    (1, 1, '2023-08-01', 'Pending', 32.98, 1),\n",
+    "    (2, 2, '2023-08-02', 'Shipped', 19.99, 2);\n",
+    "\n",
+    "INSERT INTO OrderItem (order_item_id, order_id, product_id, quantity, subtotal)\n",
+    "VALUES\n",
+    "    (1, 1, 1, 2, 39.98),\n",
+    "    (2, 1, 2, 1, 12.99),\n",
+    "    (3, 2, 1, 1, 19.99);\n"
+   ]
+  },
+  {
+   "attachments": {
```

Comment:
If you _need_ an image in your workbook, please include it as a seperate file, not inline in the notebook.

Reply:
Done. Removed the images and added to .github/workflows and .github/notebook_lists